android/local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
```

Comment:
It say that this file should **not** be commit to Git. But I think it's fine here, right?